src/components/OptimizerForm.js

```diff
@@ -82,15 +83,16 @@ export default function OptimizerForm() {
   useMemo(() => {
     let lcFn = LightConeConditionals.get(optimizerForm.getFieldsValue())
     let form = optimizerForm.getFieldsValue()
-    let defaults = lcFn.defaults()
+    let defaults = lcFn.defaults();
     let lightConeForm = form.lightConeConditionals || {}
-
     // We can't apply the form to dynamically generated elements, so we use an effect to set the form value to default
     // Only if there's a missing field
     Object.assign(defaults, lightConeForm)
-    if (Object.values(defaults).includes(undefined)) {
+  console.log('useMemo lcFn.defaults()', defaults, lcFn.defaults(), lightConeForm);
+  console.log(lcFn.defaults.valueOf());
+    // if (Object.values(defaults).includes(undefined)) {
       optimizerForm.setFieldValue('lightConeConditionals', lcFn.defaults())
-    }
+    // }
   }, [optimizerForm]);
```

Comment:
This looks kinda janky, I don't remember what the `Object.values(defaults).includes(undefined)` was for hmm